docs/systemd.md

```diff
@@ -45,7 +45,7 @@ E.g.:
 ```
 ./gen_systemd_service_file.py tezos-node
 # or
-./gen_systemd_service_file.py tezos-baker-011-PtHangz2
+./gen_systemd_service_file.py tezos-baker-012-Psithaca
 ```
 After that you'll have `.service` files in the current directory.
 
```

Comment:
The `Multiple similar systemd services` (a few lines below) also needs an update here.